backend/src/pong/game/dto/update-game.dto.ts

```diff
@@ -0,0 +1,8 @@
+import { PartialType } from '@nestjs/mapped-types';
+import { CreateGameDto } from './create-game.dto';
```

Comment:
Okay.. something makes me think this is from the tutorial.. but can't find it in that github repo (and don't want to search in the vids..)

But right now these two imports are unused
I did find this:
https://trilon.io/blog/introducing-mapped-types-for-nestjs

which would come down to something like this:
```ts
export class UpdateGameDto extends PartialType(CreateGameDto) {}
```
But I'm not sure what the benefit/use is...